examples/make_prerecorded_request_builder.rs

```diff
@@ -38,8 +38,7 @@ async fn main() -> reqwest::Result<()> {
     // That way it knows what type to deserialize the JSON into
     let response: Response = customized_request_builder.send().await?.json().await?;
 
-    let transcript = &response.results.channels[0].alternatives[0].transcript;
-    println!("{}", transcript);
+    println!("{:?}", response);
```

Comment:
Printing out entire json here for completeness and also avoids errors when using callback url

I saw some other function for callbacks but added to follow the same pattern as the other options like callback_method

Reply:
Using callbacks with the non-callback interface will error anyway, but at runtime, because the response format won't match the response struct, so it will error when it tries to deserialize the response into the struct.

Reply:
I made the whole response object Optional to support parsing both callback response and non-callback response

See: https://github.com/deepgram-devs/deepgram-rust-sdk/blob/streaming-features/src/transcription/prerecorded/response.rs#L18-L25